components-api/src/main/scala/pl/touk/nussknacker/engine/api/context/ProcessCompilationError.scala

```diff
@@ -285,18 +287,27 @@ object ProcessCompilationError {
 
   final case class DisablingManyOutputsFragment(id: String, nodeIds: Set[String]) extends ProcessCompilationError
 
-  final case class DisablingNoOutputsFragment(id: String) extends ProcessCompilationError {
+  // TODO local: check if this displays ok
```

Comment:
todo local

Reply:
same below

Reply:
thanks! btw they di not display properly but it was the same before the change. Will make these changes in other PRs